### DISCUSSION TITLE
Bem bacana desenvolver  a NFSe Nacional

### DISCUSSION THREAD

**Discussion by marcoshemann** [TadaSoftware/PyNFe#292] [Ideas]
Vai entrar em operação a emissão da NFSe por servidores federais, claro que precisa da prefeitura aderir ao convenio, mas eu acredito que a maioria deve partir para isto. Porto Alegre já aderiu.
https://www.gov.br/empresas-e-negocios/pt-br/empreendedor/servicos-para-mei/nota-fiscal/nota-fiscal-de-servico-eletronica-nfs-e

Acredito que seria interessa ter suporte a este emissor.

**Comment by treasuryesc** [comment]
Olá a todos! Nós somos da Trezzuri Tecnologia e nos interessamos pelo projeto, gostaríamos de colaborar no projeto da NFS-e nacional, além de atualizar o emissor de NF de serviço, que vimos que está usando alguns packages deprecated (pyxb e suds-jurko). 
@juniortada e @leotada vocês tem planejado desenvolver esse módulo? Como podemos colaborar?

**Comment by leogregianin** [comment]
Vou fechar essa discussão já que foi gerada issue #305 sobre o assunto.
